themes/book/app/routes/$.tsx

```diff
@@ -73,7 +73,7 @@ export function ArticlePageAndNavigation({
           <article
             ref={container}
             className="article content article-grid article-grid-gap"
-            style={{ marginTop: top + 16 }}
+            style={{ marginTop: top + 20 }}
```

Comment:
this correctly aligns the main content with the left nav -- to see it you can prepare a markdown file with a single line of text and use browser devtools to remove the frontmatter div

Reply:
Ah - I think there is also a `MAGIC_PADDING` somewhere that might need to be updated, and the comment could be improved saying where it comes from.

Do you think you could update that as part of these changes?

Reply:
It is in TableOfContence.tsx, I couldn't for the life of me figure out where that came from. :)